ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -18,6 +17,7 @@ import java.lang.Exception
  *
  * Note: **This feature is in beta. It's public API may change in future releases.**
  */
+@UnstableApi
```

Comment:
Does this add the annotation to all of the public methods in this class such as `getRecommendedPaymentMethods`? I think we'd also want to add the annotation to `ShopperInsightsRequest` and `ShopperInsightsCallback` correct?

Reply:
Ref: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-requires-opt-in/

Class level opt in makes everything in the class opt in.
<img width="957" alt="Screenshot 2024-05-01 at 9 52 49 AM" src="https://github.com/braintree/braintree_android/assets/104481964/61afbf45-8434-41c5-89b7-daab5a04ada5">

Otherwise, each of the methods requires opt-in.
<img width="1404" alt="Screenshot 2024-05-01 at 9 53 51 AM" src="https://github.com/braintree/braintree_android/assets/104481964/713573fb-b39a-4bb7-8577-bac14dc1c727">

<img width="1532" alt="Screenshot 2024-05-01 at 9 53 59 AM" src="https://github.com/braintree/braintree_android/assets/104481964/1109b7fb-fc33-4bfe-a7fd-35a02d80467d">

<img width="1531" alt="Screenshot 2024-05-01 at 9 53 40 AM" src="https://github.com/braintree/braintree_android/assets/104481964/f96e66f9-070b-40e4-a6dc-17bca62cbe01">